### PR TITLE
Complete nearly everything in item

### PR DESCRIPTION
**Pull request by diacritics-owo**
First Quilt contribution. As per the title, I've written mappings for everything in `item` except for some of `CrossbowItem` and `ItemStack`.

### CODE REVIEW

File: mappings/net/minecraft/item/Item.mapping [hunk 1]
```diff
@@ -14,6 +14,10 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 	METHOD <init> (Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;)V
 		ARG 1 settings
 	METHOD m_bdsymcbv getName ()Lnet/minecraft/unmapped/C_rdaqiwdt;
+	METHOD m_bhfktmca getDamage (Lnet/minecraft/unmapped/C_astfners;FLnet/minecraft/unmapped/C_sbxfkpyv;)F
```
Comment (OroArmor) [inline mappings/net/minecraft/item/Item.mapping]:
```suggestion
	METHOD m_bhfktmca getAttackDamage (Lnet/minecraft/unmapped/C_astfners;FLnet/minecraft/unmapped/C_sbxfkpyv;)F
```
makes it clearer its not stack damage

File: mappings/net/minecraft/item/CrossbowItem.mapping [hunk 2]
```diff
@@ -14,13 +20,28 @@ CLASS net/minecraft/unmapped/C_fatxcdqp net/minecraft/item/CrossbowItem
...
 		ARG 1 random
+	METHOD m_uhdfuuhv getSoundPitch (Lnet/minecraft/unmapped/C_rlomrsco;I)F
+		ARG 1 index
 	METHOD m_wiqutonr isCharged (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
 		ARG 0 stack
+	METHOD m_xefbydko getLoadingProgress (ILnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_usxaxydn;)F
+		ARG 0 usedTime
+	CLASS C_ldycibmo LoadingSounds
+		METHOD m_dvaiaquf (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+			ARG 0 loadingSounds
```
Comment (OroArmor) [inline mappings/net/minecraft/item/CrossbowItem.mapping]:
```suggestion
			ARG 0 sounds
```

**Review by OroArmor — CHANGES_REQUESTED** [review]
Some small changes

**Comment by diacritics-owo** [comment]
I've applied the suggestions! Let me know if there's anything else

**Review by supersaiyansubtlety — APPROVED** [review]
~~why would Mojang do this to us in particular `register("banner_white", "white_banner"...`~~
looks good

**Review by OroArmor — APPROVED** [review]

**Pull request merged**